content/en/docs/tasks/administer-cluster/storage-object-in-use-protection.md

```diff
@@ -216,7 +216,7 @@ spec:
 - Verify that the scheduling of the second pod fails with the below warning:
 
 ```shell
-Warning  FailedScheduling  18s (x4 over 21s)  default-scheduler  persistentvolumeclaim "slzc" is being deleted
+Warning  FailedScheduling  18s (x4 over 21s)  default-scheduler persistentvolumeclaim "slzc" is being deleted
```

Comment:
Since this is a warning message, perhaps the extra space is intended?

Reply:
Do I need to revise it back?

Reply:
As the space does not change the meaning of the warning, I doubt that it is a problem. It is okay.